drivers/d3d12/rendering_device_driver_d3d12.cpp

```diff
@@ -96,11 +104,6 @@ static const D3D12_RANGE VOID_RANGE = {};
 static const uint32_t ROOT_CONSTANT_REGISTER = GODOT_NIR_DESCRIPTOR_SET_MULTIPLIER * (RDD::MAX_UNIFORM_SETS + 1);
 static const uint32_t RUNTIME_DATA_REGISTER = GODOT_NIR_DESCRIPTOR_SET_MULTIPLIER * (RDD::MAX_UNIFORM_SETS + 2);
 
-#ifdef DEV_ENABLED
-//#define DEBUG_COUNT_BARRIERS
-#define CUSTOM_INFO_QUEUE_ENABLED 0
-#endif
-
```

Comment:
I'd keep these as they hint about dev-only features that enable certain code paths.

Reply:
It's moved to the header not removed, since defines in it are used there as well (to avoid declaring unused variables).